src/Tizen.NUI.Components/Controls/ItemsView/ItemsLayouter.cs

```diff
@@ -0,0 +1,281 @@
+﻿/* Copyright (c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using Tizen.NUI.BaseComponents;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Tizen.NUI.Binding;
+
+namespace Tizen.NUI.Components
+{
+    /// <summary>
+    /// [Draft] Defalt layout manager for CollectionView.
+    /// Lay out ViewItem and recycle ViewItem.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public abstract class ItemsLayouter : ICollectionChangedNotifier
+    {
+        /// <summary>
+        /// Container which contains ViewItems.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected View Container{get;set;}
+        /// <summary>
+        /// Parent ItemsView.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        protected ItemsView ItemsView{get;set;}
+
+        /// <summary>
+        /// The last scrolled position which is calculated by ScrollableBase. The value should be updated in the Recycle() method.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected float PrevScrollPosition{get;set;}
+
+        /// <summary>
+        /// First index of visible items.
+        /// </summary>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected int FirstVisible = -1;
```

Comment:
protected field는 허용되지 않기 때문에 Property로 변경 부탁드립니다.
```suggestion
        protected int FirstVisible { get; set; } = -1;
```